db/migrate/20170606115344_change_lookup_key_path_to_text.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeLookupKeyPathToText < ActiveRecord::Migration
+  def change
+    remove_index :lookup_keys, :path
```

Comment:
I was really surprised to see we don't use the index at all, but you are correct...